smartcontractkit/external-adapters-js
# Feature/31870 - Downgrade all node versions to 16.x

## Closes #31870

## Description
Downgrade node to version 16.14.0 for all processes.
......

## Changes

- Use Node 16.x for all github actions/workflows that previously used Node 17.x
- Remove `NODE_OPTIONS=--openssl-legacy-provider` in a number of areas, including the `renvm-address-set` adapter. This should not be needed for `16.x`.

## Steps to Test

1. Run locally & run tests locally
2. Run in Docker
3. Run a deployment and make sure all tests pass

## Quality Assurance
